api/proto/teleport/legacy/types/types.proto

```diff
@@ -2428,6 +2442,13 @@ message AccessRequestFilter {
   string User = 2 [(gogoproto.jsontag) = "user,omitempty"];
   // RequestState filters for requests in a specific state.
   RequestState State = 3 [(gogoproto.jsontag) = "state,omitempty"];
+  // SearchKeywords is a list of search keywords to match against resource field values.
+  repeated string SearchKeywords = 4 [(gogoproto.jsontag) = "search,omitempty"];
```

Comment:
Can we detail a little more how the search works?
Does it return results that contain at least one of the keywords? Or only requests that have all the keywords?

Reply:
Sorry about that, this field was copy-paste from another filter where these exists. From Resource `MatchSearch`
```
// MatchSearch goes through select field values from a resource
// and tries to match against the list of search values, ignoring case and order.
// Returns true if all search vals were matched (or if nil search vals).
// Returns false if no or partial match (or nil field values).
```

Reply:
Can we add that to the comment?

Reply:
Sure thing!